server/config.js

```diff
@@ -0,0 +1,9 @@
+
+module.exports = {
+  config: {
+    env: process.env.NODE_ENV || 'development',
+    port: process.env.PORT || 8080,
+    host: 'localhost:',
+    dataBase: 'mongodb://wg:1991993@ds157559.mlab.com:57559/cookbook-recipes'
+  }
+}
```

Comment:
If we are going to be using environmental variables inside the config files, why not just use something like (dotenv)[https://www.npmjs.com/package/dotenv]. Then we can store all of our environmental variables in a file that will be gitignored.